cluster/gce/gci/configure-helper.sh

```diff
@@ -156,6 +156,7 @@ function main() {
   reset-motd
   prepare-mounter-rootfs
   modprobe configs
+  start-prometheus &  #####start prometheus
```

Comment:
this one only for main master, if you have partitioned server, you need double check if your partitioned server also has this started.